content/blog/how-to-monitor-hpe-compute-ops-management.md

```diff
@@ -0,0 +1,362 @@
+
+HPE Compute Ops Management REST API uses the OAuth 2.0 authentication based on the client credential, which generates a limited lifetime access token.
+
+The access token is a long string in the form of a JSON Web Token that is signed using RS256 algorithm. The access token must be added into the HTTP header with keyword "Authorization: Bearer {token}" for any REST API request. 
+
+For information about how to generate an access token for Compute Ops Management, please refer to [this link ](https://developer.greenlake.hpe.com/docs/greenlake/guides/public/authentication/authentication/)for more details.
+
+Only a few resource metrics are currently supported by HPE Compute Ops Management via the RESTful API, but things will change quickly in the coming months. Today, the only metric available is the carbon footprint report but many other resources are available to create nice Grafana dashboards such as data related to the number of servers, health of servers, service packs, groups, etc. 
+
+# Grafana Infinity plugin
+
+There are several Grafana plugins that support data collection via the REST API (e.g. Infinity, [JSON](https://grafana.com/grafana/plugins/simpod-json-datasource/), [JSON API](https://grafana.com/grafana/plugins/marcusolsson-json-datasource/)) but [Infinity ](https://grafana.com/grafana/plugins/yesoreyeram-infinity-datasource/)has the great advantage of offering an advanced query language that is essential for manipulating JSON data into a suitable format that Grafana can understand. This language is called [UQL](https://sriramajeyam.com/grafana-infinity-datasource/wiki/uql/), Infinity's unstructured query language.
+
+UQL is not simple at first glance but I will provide examples in this blog. With UQL, you can customize the results you need regardless of the json format returned by the API.
+
+A UQL query can be formed with a list of commands joined by |. Most of the time, fields are referenced in double quotes and string values are referenced in single quotes as shown below:
+
+![](/img/2022-10-19-16_33_28-hpe-software-‎-onenote-for-windows-10.png)
+
+The following diagram describes the different components of the solution:
+
+![](/img/2022-10-19-16_15_11-lj-synergy-composable-fabric.pptx-powerpoint.png)
+
+## Pros and Cons about this solution
+
+Pros:
+
+* Lightweight solution as it only requires Grafana and an easily installable plugin
```

Comment:
Recommend rewording first bullet to read:
"**A** lightweight solution **that** only requires Grafana and an easily installable plugin"